acceptance/testdata/builder/container/cnb/buildpacks/hello_world/0.0.1/buildpack.toml

```diff
@@ -0,0 +1,19 @@
+# Buildpack API version
+api = "0.2"
+
+# Buildpack ID and metadata
+[buildpack]
+id = "hello_world"
+version = "0.0.1"
+name = "Hello World Buildpack"
+homepage = "https://github.com/buildpacks/samples/tree/main/buildpacks/hello-world"
+
+# Stacks that the buildpack will work with
+[[stacks]]
+id = "io.buildpacks.samples.stacks.bionic"
+
+[[stacks]]
+id = "io.buildpacks.samples.stacks.alpine"
+
+[[stacks]]
+id = "io.buildpacks.stacks.bionic"
```

Comment:
I think this is all not used by the tests, and could be removed.

Reply:
I deleted most of parts of buildpack.toml files please check again. We need some keys from this file. When i delete this file completely tests are failing.